game/image.hh

```diff
@@ -43,7 +44,7 @@ struct Bitmap {
 	}
 	unsigned char const* data() const { return ptr ? ptr : &buf[0]; }
 	unsigned char* data() { return ptr ? ptr : &buf[0]; }
-
+	void copyFromCairo(cairo_surface_t* surface);
```

Comment:
Just a thinking: Looks like a bit of SRP violation because bitmap is an abstract image. In theory we could just move it to the separate method and all will be better. Or even change loadSVG so that it will create a new image and not try to reuse old. 

But I think it's ok to leave things as they are. It's just a refactoring proposal, not development.